src/kernel/src/fs/path.rs

```diff
@@ -286,11 +287,60 @@ impl From<VPathBuf> for String {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum LookupOp {
```

Comment:
This one should not be here.

Reply:
Seems like to missed this one. It should be in the root of `fs` instead of `path`.